extension/source/Home/Wallet/Wallets/Recovery/StepTwoWalletCreation.tsx

```diff
@@ -0,0 +1,76 @@
+import { CaretRight, CircleNotch, CopySimple } from 'phosphor-react';
+import { FunctionComponent, useEffect, useState } from 'react';
+import Button from '../../../../components/Button';
+import { useQuill } from '../../../../QuillContext';
+
+const StepTwoWalletCreation: FunctionComponent<{
+  onComplete: () => void;
+  setWalletToParent: (address: string) => void;
+}> = ({ onComplete, setWalletToParent }) => {
+  const [walletAddress, setWalletAddress] = useState<string>('');
+  const [loading, setLoading] = useState<boolean>(true);
+
+  const { rpc } = useQuill();
+
+  useEffect(() => {
+    const createWallet = async () => {
+      setLoading(true);
+      const { address, privateKey } = await rpc.createTempAccount();
+      setWalletAddress(address);
+      setWalletToParent(privateKey);
+      setLoading(false);
+    };
+
+    createWallet();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are deps disabled here? Should just be `rpc`.

Reply:
I sometimes don't understand these deps. Like it's not possible to change some deps without explicit reload which is gonna rerender anyway.

But yeah, let's add rpc here.

Reply:
Useful comment on this issue with links for further reading from Dan Abramov
https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528